appveyor.yml

```diff
@@ -36,4 +36,4 @@ after_build:
 
 test_script:
   - set CGO_ENABLED=1
-  - go run build\ci.go test -vet -coverage
+  - go run build\ci.go test -vet -coverage -misspell
```

Comment:
The check doesn't need to run on AppVeyor and Travis. Removing it from AppVeyor would remove the workaround.